mod/05_standard/set/self/account_links.rb

```diff
@@ -5,8 +5,9 @@
     #ENGLISH
     links = []
     if Auth.signed_in?
-      links << link_to_page( Auth.current.name, nil, :id=>'my-card-link' )
+      links << build_link( Auth.current.cardname, nil, :id=>'my-card-link' )
```

Comment:
I still prefer the two arg approach, fwiw.  It's weird, imo, to have all the explicit nil calls.  No need to do that now...